pkg/generator/templates.go

```diff
@@ -255,32 +255,32 @@ required = [
 
 [[override]]
   name = "k8s.io/code-generator"
-  # revision for tag "kubernetes-1.10.1"
-  revision = "7ead8f38b01cf8653249f5af80ce7b2c8aba12e2"
+  # revision for tag "kubernetes-1.11.2"
+  revision = "6702109cc68eb6fe6350b83e14407c8d7309fd1a"
 
 [[override]]
   name = "k8s.io/api"
-  # revision for tag "kubernetes-1.10.1"
-  revision = "73d903622b7391f3312dcbac6483fed484e185f8"
+  # revision for tag "kubernetes-1.11.2"
+  revision = "2d6f90ab1293a1fb871cf149423ebb72aa7423aa"
 
 [[override]]
   name = "k8s.io/apiextensions-apiserver"
-  # revision for tag "kubernetes-1.10.1"
-  revision = "4347b330d0ff094db860f2f75fa725b4f4b53618"
+  # revision for tag "kubernetes-1.11.2"
+  revision = "408db4a50408e2149acbd657bceb2480c13cb0a4"
 
 [[override]]
   name = "k8s.io/apimachinery"
-  # revision for tag "kubernetes-1.10.1"
-  revision = "302974c03f7e50f16561ba237db776ab93594ef6"
+  # revision for tag "kubernetes-1.11.2"
+  revision = "103fd098999dc9c0c88536f5c9ad2e5da39373ae"
 
 [[override]]
   name = "k8s.io/client-go"
-  # revision for tag "kubernetes-1.10.1"
-  revision = "989be4278f353e42f26c416c53757d16fcff77db"
+  # revision for tag "kubernetes-1.11.2"
+  revision = "1f13a808da65775f22cbf47862c4e5898d8f4ca1"
 
 [[override]]
   name = "sigs.k8s.io/controller-runtime"
-  revision = "60bb251ad86f9b313653618aad0c2c53f41a6625"
+  version = "v0.1.3"
```

Comment:
Turn this back to `60bb251ad86f9b313653618aad0c2c53f41a6625`.
That's the commit that has the controller-runtime supporting a namespaced cache. We don't have it on the master branch yet but we'll change that.

Reply:
yes, u r right.

Reply:
Do you mean `ca13e86844d4195a5c95096de1f0d3610db1a8a3` https://github.com/kubernetes-sigs/controller-runtime/pull/136/commits/ca13e86844d4195a5c95096de1f0d3610db1a8a3?

Reply:
actually we can use https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.1.4 instead.

Reply:
Yeah anything that has the namespace fix merged. We can use `v0.1.4`. You'll have to update the scaffold template for Gopkg.toml as well because that's the one that is actually used by a project.

Reply:
I update the template for Gopkg.toml .